plumbing/format/commitgraph/encoder.go

```diff
@@ -18,7 +18,7 @@ type Encoder struct {
 
 // NewEncoder returns a new stream encoder that writes to w.
 func NewEncoder(w io.Writer) *Encoder {
-	h := hash.New(hash.CryptoType)
+	h := hash.New(crypto.SHA1)
```

Comment:
Let's put a todo here to add support for format object.

```suggestion
	// TODO: Support passing an ObjectFormat instead of hardcoding SHA1
	h := hash.New(crypto.SHA1)
```